docs/source/resources/troubleshooting.md

```diff
@@ -55,3 +59,21 @@ def _memory_usage(df):
     size += df.index.memory_usage(deep=True)
     return size
 ```
+
+### 2. Initializing a Dask CUDA Cluster
+
+Even if you execute the NVTabular workflow with only a single GPU, using a dask CUDA cluster can optimize your memory usage. If there is no CUDA cluster initialized, NVTabular executes the workflow on workers of a dask cluster. Dask CUDA cluster provides different memory spilling strategies for GPUs and in our experiment, we could resolve OOM errors by initializing a dask CUDA cluster, first. It is easy to provide NVTabular with a dask CUDA cluster by using `LocalCUDACluster`.
+
+```python
+from dask_cuda import LocalCUDACluster
+from dask.distributed import Client
+
+import nvtabular as nvt
+
+cluster = LocalCUDACluster()
+client = Client(cluster)
+
+features = ['col'] >> nvt.ops.Normalize()
+
+workflow = nvt.Workflow(features, client=client)
+```
```

Comment:
we should add something about cleaning up the client (```client.shutdown()```) after running the workflow

Reply:
Is it `client.shutdown()` or `client.close()`?

I normally use client.close()

Reply:
After all work is completed, you would do something like:
```python
client.shutdown()
client.close()
```

Reply:
Added both shutdown and close